PreFreeSurfer/scripts/T2wToT1wDistortionCorrectAndReg.sh

```diff
@@ -176,6 +176,37 @@ Modalities="T1w T2w"
 
 log_Msg "START"
 
+verbose_echo "  "
+verbose_red_echo " ===> Running T2WToT1wDistortionCorrectAndReg"
+verbose_echo "  "
+verbose_echo "  Parameters"
+verbose_echo "                           WD          (workingdir): $WD"
+verbose_echo "                     T1wImage                  (t1): $T1wImage"
+verbose_echo "                T1wImageBrain             (t1brain): $T1wImageBrain"
+verbose_echo "                     T2wImage                  (t2): $T2wImage"
+verbose_echo "                T2wImageBrain             (t2brain): $T2wImageBrain"
+verbose_echo "           MagnitudeInputName             (fmapmag): $MagnitudeInputName"
+verbose_echo "               PhaseInputName           (fmapphase): $PhaseInputName"
+verbose_echo "                GEB0InputName (fmapgeneralelectric): $GEB0InputName"
+verbose_echo "                           TE            (echodiff): $TE"
+verbose_echo "  SpinEchoPhaseEncodeNegative          (SEPhaseNeg): $SpinEchoPhaseEncodeNegative"
+verbose_echo "  SpinEchoPhaseEncodePositive          (SEPhasePos): $SpinEchoPhaseEncodePositive"
+verbose_echo "               SEEchoSpacing        (seechospacing): $SEEchoSpacing"
+verbose_echo "                  SEUnwarpDir         (seunwarpdir): $SEUnwarpDir"
+verbose_echo "             T1wSampleSpacing       (t1sampspacing): $T1wSampleSpacing"
+verbose_echo "             T2wSampleSpacing       (t2sampspacing): $T2wSampleSpacing"
+verbose_echo "                    UnwarpDir           (unwarpdir): $UnwarpDir"
+verbose_echo "               OutputT1wImage                 (ot1): $OutputT1wImage"
+verbose_echo "          OutputT1wImageBrain            (ot1brain): $OutputT1wImageBrain"
+verbose_echo "           OutputT1wTransform             (ot1warp): $OutputT1wTransform"
+verbose_echo "               OutputT2wImage                 (ot2): $OutputT2wImage"
+verbose_echo "           OutputT2wTransform             (ot2warp): $OutputT2wTransform"
+verbose_echo "         DistortionCorrection              (method): $DistortionCorrection"
+verbose_echo "                  TopupConfig         (topupconfig): $TopupConfig"
+verbose_echo "     GradientDistortionCoeffs            (gdcoeffs): $GradientDistortionCoeffs"
```

Comment:
Add value of UseJacobian to the reported list.

Reply:
Ok.